roles/sonarqube/templates/rhsso-sonar4-client.yml.j2

```diff
@@ -18,17 +18,18 @@
       clientAuthenticatorType: client-secret
       publicClient: false
       redirectUris:
-        - 'https://sonarqube-sample-sonar-example.apps.{{ subdomain }}/*'
-      rootUrl: 'https://sonarqube-sample-sonar-example.apps.{{ subdomain }}/'
+        - 'https://sonarqube-devsecops.apps.{{ subdomain }}/oauth2/callback/saml'
```

Comment:
We should use `{{ sonarqube_project_name }}` here

Reply:
@andykrohg  - good catch, thanks. Fixed !